python/tests/integration/connectors/memory/test_postgres.py

```diff
@@ -1,214 +1,206 @@
 # Copyright (c) Microsoft. All rights reserved.
 
-import time
+import uuid
```

Comment:
nit: Perhaps rename this file to `test_postgres_store.py`?

Reply:
I was basing the naming on the other stores; it looks like the pattern is `test_redis.py`, `test_pinecone.py` etc. It looks like mostly the integration tests for specific memory connectors are for MemoryStore, whereas the new VectorStore tests are consolidated into `test_vector_collections.py`. I wanted to make Postgres-specific integration tests so figured I would replace the existing `test_postgres.py` with the new vector store integration tests, and move the old memory store tests to `test_postgres_memory_store.py`.

That said, I'm happy to name it `test_postgres_store.py` to establish a new naming convention for connector-specific vector store integration tests. Sound good?

Reply:
I think eventually we want to have tests based on scenarios, not per connector (similar to the AI connectors). `test_postgres.py` and `test_postgres_memory_store.py` sound good to me now.